src/psyclone/dynamo0p3.py

```diff
@@ -75,10 +75,7 @@
     CONTINUOUS_FUNCTION_SPACES
 
 # Valid any_space metadata (general FS, treated as continuous)
-# TODO (issue #455): add any_space_10
-VALID_ANY_SPACE_NAMES = ["any_space_1", "any_space_2", "any_space_3",
-                         "any_space_4", "any_space_5", "any_space_6",
-                         "any_space_7", "any_space_8", "any_space_9"]
+VALID_ANY_SPACE_NAMES = ["any_space_{0}".format(x+1) for x in range(10)]
```

Comment:
Much neater. I think the comment above should probably be changed from "treated as continuous" to "could be continuous or discontinuous".

Reply:
Chamged, thanks for the suggestion.